### ISSUE TITLE
about acdc_dataset

### ISSUE THREAD

**Issue by Yss83** [HuCaoFighting/Swin-Unet#83]
I know you can't  share the preprocessed acdc_dataset, can you tell me which patients you use as the training set and which patients do the test set. Thank you in advance.
